jax/core.py

```diff
@@ -353,6 +345,8 @@ def write(v, val):
       write(eqn.outvars[0], ans)
   return map(read, jaxpr.outvars)
 
+initial_to_final_param_rules: Dict[Primitive, Callable] = {}
```

Comment:
This may be counter to the jax philosophy of confusing our enemies, but consider a comment.